src/thing/thing.ts

```diff
@@ -228,7 +228,7 @@ export function removeThing<Dataset extends SolidDataset>(
   return newSolidDataset;
 }
 
-function withChangeLog<Dataset extends SolidDataset>(
+export function withChangeLog<Dataset extends SolidDataset>(
```

Comment:
If you're exporting this, make sure to rename it to `internal_withChangelog` and move it to `thing.internal.ts`, to avoid people reaching in and using this without API stability guarantees.

Reply:
5ad3ec10eda88b7fa7a735c71cb127706140e7bb

Reply:
That didn't move it to `thing.internal.ts` :)